### PR TITLE
docs(topic operator): describes the tuning for topic operator batching mechanism

### PR DESCRIPTION
**Pull request by PaulRMellor**
**Documentation**

New section to describe how to  tune the batching mechanism used by the UTO using operator config properties.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

File: documentation/modules/operators/con-tuning-topic-request-batches.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+//
+// assembly-using-the-topic-operator.adoc
+
+[id='con-tuning-topic-request-batches-{context}']
+= (Preview) Tuning request batches for topic operations
+
+In unidirectional mode, the Topic Operator uses the request batching capabilities of the Kafka Admin API for operations on topic resources. 
+You can fine-tune the batching mechanism using the following operator configuration properties:
+
+* `STRIMZI_MAX_QUEUE_SIZE` to set the maximum size of the request batching queue.
```
Comment (fvaleri) [inline documentation/modules/operators/con-tuning-topic-request-batches.adoc:11]:
```suggestion
* `STRIMZI_MAX_QUEUE_SIZE` to set the maximum size of the topic event queue. 
```

File: documentation/modules/operators/con-tuning-topic-request-batches.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+[id='con-tuning-topic-request-batches-{context}']
+= (Preview) Tuning request batches for topic operations
+
+In unidirectional mode, the Topic Operator uses the request batching capabilities of the Kafka Admin API for operations on topic resources. 
+You can fine-tune the batching mechanism using the following operator configuration properties:
+
+* `STRIMZI_MAX_QUEUE_SIZE` to set the maximum size of the request batching queue. 
+The default value is 1024.
+* `STRIMZI_MAX_BATCH_SIZE` to set the maximum number of requests allowed in a single batch.
```
Comment (fvaleri) [inline documentation/modules/operators/con-tuning-topic-request-batches.adoc:13]:
```suggestion
* `STRIMZI_MAX_BATCH_SIZE` to set the maximum number of topic events allowed in a single batch. 
```

File: documentation/modules/operators/con-tuning-topic-request-batches.adoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+[id='con-tuning-topic-request-batches-{context}']
+= (Preview) Tuning request batches for topic operations
+
+In unidirectional mode, the Topic Operator uses the request batching capabilities of the Kafka Admin API for operations on topic resources. 
+You can fine-tune the batching mechanism using the following operator configuration properties:
+
+* `STRIMZI_MAX_QUEUE_SIZE` to set the maximum size of the request batching queue. 
+The default value is 1024.
+* `STRIMZI_MAX_BATCH_SIZE` to set the maximum number of requests allowed in a single batch. 
+The default value is 100 requests for operations to create, update, or delete topics.
```
Comment (fvaleri) [inline documentation/modules/operators/con-tuning-topic-request-batches.adoc:14]:
```suggestion
The default value is 100.
```

File: documentation/modules/operators/con-tuning-topic-request-batches.adoc [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+* `MAX_BATCH_LINGER_MS` to specify the time to wait for a request batch to accumulate `STRIMZI_MAX_BATCH_SIZE` items before processing.
```
Comment (fvaleri) [inline documentation/modules/operators/con-tuning-topic-request-batches.adoc:15]:
```suggestion
* `MAX_BATCH_LINGER_MS` to specify the max time to wait for a batch to accumulate items before processing. 
```

**Review by fvaleri — CHANGES_REQUESTED** [review]
Thanks. Looks good, just some suggestions.

**Review by ppatierno — APPROVED** [review]
LGTM.

**Review by fvaleri — APPROVED** [review]

**Pull request merged**